BDArmory/Settings/CompSettings.cs

```diff
@@ -48,27 +53,15 @@ public static void Load()
 
             foreach (ConfigNode.Value fieldNode in settings.values)
             {
-                var field = typeof(BDArmorySettings).GetField(fieldNode.name, BindingFlags.Public | BindingFlags.Static | BindingFlags.DeclaredOnly);
-                if (field == null)
-                {
-                    Debug.LogError($"[BDArmory.CompSettings]: Unknown field {fieldNode.name} when loading AI/WM override settings.");
-                    continue;
-                }
-                var fieldValue = BDAPersistentSettingsField.ParseValue(field.FieldType, fieldNode.value);
-                ComPOverrides[fieldNode.name] = fieldValue; // Add or set the override.
+                var fieldValue = float.Parse(fieldNode.value);
```

Comment:
This should use
```csharp
if (float.TryParse(fieldNode.value, out float fieldValue))
```
in case the `fieldNode.value` isn't parseable as a `float`, in which case it should log an error.